packages/host/app/components/matrix/room-message.gts

```diff
@@ -108,7 +108,7 @@ export default class Room extends Component<Signature> {
             </Button>
             <div
               class='monaco-container'
-              {{setCssVar monaco-container-height=this.monacoContainerHeight}}
+              {{this.viewCodePanelModifier}}
```

Comment:
Does it make sense to describe what the modifier does vs where it’s used?

Reply:
Any suggestions? `panelHeightAndScrollModifier`? I'm terrible with naming

Reply:
`scrollBottomIntoView`? 🤔